src/Orchard.Web/Modules/Orchard.DynamicForms/Drivers/NumFieldElementDriver.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Globalization;
+using Orchard.DynamicForms.Elements;
+using Orchard.Forms.Services;
+using Orchard.Layouts.Framework.Display;
+using Orchard.Layouts.Framework.Drivers;
+using Orchard.Layouts.Helpers;
+using Orchard.Layouts.Services;
+using Orchard.Tokens;
+using DescribeContext = Orchard.Forms.Services.DescribeContext;
+
+namespace Orchard.DynamicForms.Drivers {
+    public class NumFieldElementDriver : FormsElementDriver<NumField>{
```

Comment:
You're right. Maybe because I had (or I thought I could have) an ambiguous reference with the already existing NumericField class in Orchard.Fields (referenced in Orchard.DynamicForms).

Just seen that seems to be the same with the TextField. So, I think you're right and, if this PR is re-open, I will do this update.

Thanks